hail/src/main/scala/is/hail/expr/types/physical/PCanonicalStruct.scala

```diff
@@ -133,6 +133,9 @@ final case class PCanonicalStruct(fields: IndexedSeq[PField], required: Boolean
   def loadField(offset: Code[Long], fieldName: String): Code[Long] =
     loadField(offset, fieldIdx(fieldName))
 
+  def loadField(offset: Long, fieldName: String): Long =
```

Comment:
Not sure that we want this, we're just missing the interpreted function and it makes writing a basic test of this functionality easier.